docs/configuration/settings.rst

```diff
@@ -569,6 +569,162 @@ You can set ``account_create_principals`` if you want to limit account creation
+
+.. note::
+
+    Both the account validation and password reset need a properly configured
+    smtp server.
+    To use a debug or testing mailer you may use the ``mail.mailer = debug`` or
+    ``mail.mailer = testing`` settings. Refer to
+    `pyramid_mailer's configuration <https://docs.pylonsproject.org/projects/pyramid_mailer/en/latest/#configuration>`_.
+
+You can restrict the email addresses allowed using the
+``account_validation.email_regexp`` setting, and the delay for which the
+activation key will be valid:
+
+.. code-block:: ini
+
+    # Set the regular expression used to validate a proper email address.
+    kinto.account_validation.email_regexp = "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$"
+    # Set the "time to live" for the activation key stored in the cache. After that
+    # delay the account won't be activable anymore.
+    kinto.account_validation.validation_key_cache_ttl_seconds = 604800  # 7 days in seconds.
+
+Once :ref:`created <accounts-create>`, the account will need to be activated
+before being able to authenticate, using the ``validate`` endpoint and the
+``activation-key`` sent by email.
+
+.. sourcecode:: bash
+
+    $ echo '{"data": {"id": "bob@example.com", "password": "azerty123"}}' | http POST http://localhost:8888/v1/accounts --verbose
+
+If the user was created, an email was sent to the user with the activation key,
+which needs to be POSTed to the ``validate`` endpoint.
+
+Example email:
+
+::
+
+    Content-Type: text/plain; charset="us-ascii"
+    MIME-Version: 1.0
+    Content-Transfer-Encoding: quoted-printable
+    From: admin@example.com
+    Subject: activate your account
+    To: bob@example.com
+    Content-Disposition: inline
+
+    2fe7a389-3556-4c8f-9513-c26bfc5f160b
+
+It is the responsability of the administrator to tell the mail recipient how to
+validate the account by modifying the email body template in the settings.
+
+This could be done by providing a link to a webapp that displays a form to the
+user with a call to action to validate the user, which will POST the activation
+key to the ``validate`` endpoint.
+
+Or the email could explain how to copy the activation code and paste it in some
+settings window.
+
+The templates for the email subject and body can be customized:
+
+.. code-block:: ini
+
+    kinto.account_validation.email_subject_template = "Account activation"
+    kinto.account_validation.email_body_template = "Hello {id},\n you can now activate your account using the following link:\n {form-url}{activation-key}"
+
+... and they will be ``String.format``-ted with the content of the user, an
+optional additional ``email-context`` provided alongside the user object, and
+the ``activation-key``:
+
+.. sourcecode:: bash
+
+    $ echo '{"data": {"id": "bob@example.com", "password": "azerty123", "email-context": {"name": "Bob Smith", "form-url": "https://example.com/validate/"}}}' | http POST http://localhost:8888/v1/accounts --verbose
+
```

Comment:
This docs about `email-context` belong to API docs, *settings* docs are about configuration not API usage :wink:

Reply:
Oh wait, I understood the opposite from one of your previous review, sorry about that will move it back to API

Reply:
I may have said something confusing. 

I think we can consider: API docs are for web developers who interact with the API, and Settings docs are for admins who install and run the service